smriprep/cli/run.py

```diff
@@ -176,8 +176,10 @@ def _warn_redirect(message, category, filename, lineno, file=None, line=None):
     warnings.showwarning = _warn_redirect
 
     # Precedence: --fs-license-file, $FS_LICENSE, default_license
-    license_file = opts.fs_license_file or os.getenv('FS_LICENSE')
-    if not check_valid_fs_license(lic=license_file):
+    if opts.fs_license_file is not None:
+        os.environ["FS_LICENSE"] = opts.fs_license_file
```

Comment:
Stringify and make absolute:

```suggestion
        os.environ["FS_LICENSE"] = os.path.abspath(opts.fs_license_file)
```